pages/plugins/graphql.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: default
+title: GraphQL documentation
+rightmenu: true
+permalink: /plugins/graphql
+---
+
+<div id="spy-nav" class="right-menu" markdown="1">
+* [Getting Started](#getting-started)
+* [Create Query](#create-query)
+* [Create Command](#create-command)
+* [Create Subscription](#create-subscription)
+* [Pass context](#pass-context)
+</div>
+
+<h1 class="no-margin-top">GraphQL Plugin</h1>
+
+This plugin allows implementing the [GraphQL specification](https://graphql.org)
+with some easy steps.
+
+## Getting Started
+
+Add the dependencies:
+
+```xml
+<dependency>
```

Comment:
Aren't these included if you add `javalin-graphql` ?

Reply:
I have changed it. Is it okay?

Reply:
Looks good!